docs/UserGuide.md

```diff
@@ -115,7 +117,7 @@ All of CCA Manager's functionality is accessed via commands. This includes both
 
 ### Upon Booting Up
 
-When you first boot up the app, the app will contain some example contacts for you to get a feel of using CCA Manager's functionalities.
+When you first boot up the app, the app will contain some example contacts for you to get a feel of using CCA Manager's [_functionalities_](#Glossary).
```

Comment:
One of the feedback we got during UG review is to rename functionalities to functions. Lets remove this from glossary and rename it.
```suggestion
When you first boot up the app, the app will contain some example contacts for you to get a feel of using CCA Manager's functions.
```